apps/web/app/api/v1/management/responses/route.ts

```diff
@@ -0,0 +1,51 @@
+import { responses } from "@/lib/api/response";
+import { createResponse, getEnvironmentResponses } from "@formbricks/lib/services/response";
+import { authenticateRequest } from "@/app/api/v1/auth";
+import { NextResponse } from "next/server";
+import { TResponse, ZResponseInput } from "@formbricks/types/v1/responses";
+import { transformErrorToDetails } from "@/lib/api/validator";
+import { DatabaseError } from "@formbricks/types/v1/errors";
+
+export async function GET(request: Request) {
+  try {
+    const authentication = await authenticateRequest(request);
+    if (!authentication) {
+      responses.notAuthenticatedResponse();
+    }
+    const responseArray = await getEnvironmentResponses(authentication.environmentId!);
+    return responses.successResponse(responseArray);
+  } catch (error) {
+    if (error instanceof DatabaseError) {
+      return responses.badRequestResponse(error.message);
+    }
+    throw error;
+  }
+}
+
+export async function POST(request: Request): Promise<NextResponse> {
```

Comment:
Please remove this functions as we don't want to allow users to create new responses via the management API (we use the client API for this)